src/web/install_dependencies.sh

```diff
@@ -9,5 +9,29 @@ fi
 
 echo 🟢 Installing npm dependencies to: $NODE_MODULES_DIR
 
+if [ -d "${NODE_MODULES_DIR}node_modules" ]; then
```

Comment:
Not sure if comments are a thing in scripting...if they are, I think it could be helpful to say why we are removing the node modules or specifically the ol-layerswitcher module in particular

Reply:
Yes, I will update and they are a thing.